src/main/java/tech/sourced/siva/Index.java

```diff
@@ -3,14 +3,18 @@
 import java.nio.file.FileSystems;
 import java.nio.file.PathMatcher;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Probably a knit, but I'm not sure which styleguide is used here and usage of the wildcard imports is [discouraged](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports) by some.

In my experience, it's quite useful not to have guess where exactly things do come from.

Reply:
Imports optimization is done automatically by intellij, should I change it anyway?